nupm/utils/dirs.nu

```diff
@@ -90,7 +75,7 @@ export def cache-dir [--ensure]: nothing -> path {
 }
 
 export def tmp-dir [subdir: string, --ensure]: nothing -> path {
-    let d = $env.NUPM_TEMP
+    let d = $env.nupm.temp
```

Comment:
this is giving me a failure in `tests [ with-test-env ]`, scratching my head at this:

Error: nu::shell::column_not_found

  x Cannot find column 'nupm'
    ,-[/Users/mkatychev/.config/nupm/nupm/utils/dirs.nu:80:13]
 79 | export def tmp-dir [subdir: string, --ensure]: nothing -> path {
 80 |     let d = $env.nupm.temp
    :             ^^^^^^^|^^^^^^|
    :                    |      `-- value originates here
    :                    `-- cannot find column 'nupm'
 81 |         | path join $subdir
    `----

Ran 1 tests. 0 succeeded, 1 failed.
Error:   × some tests failed
```

Reply:
@Rydwxz @kubouch   I've managed to reproduce the issue, it's quite strange, take a look here:
https://github.com/nushell/nushell/issues/16036

Reply:
Workaround implemented in https://github.com/nushell/nupm/pull/122/commits/1f1249098165c3d1ab25446642e4c769a627225a